src/components/shared/Fields/InputStatus/HookForm/InputStatus.tsx

```diff
@@ -45,8 +44,7 @@ const HookFormInputStatus = ({
     <Element
       className={getMainClasses(appearance, styles, {
         error: !!error && !isLoading,
-        hidden:
-          (!text && !isLoading) || (!!error && !isNil(touched) && !touched),
```

Comment:
This `isNil` was causing the status to appear before the field had been touched.